consensus/core/src/core_thread.rs

```diff
@@ -86,7 +88,11 @@ pub(crate) enum CoreError {
 #[allow(unused)]
 impl CoreThreadDispatcher {
     pub fn start(core: Core, context: Arc<Context>) -> (Self, CoreThreadDispatcherHandle) {
-        let (sender, receiver) = mpsc::channel(32);
+        let (sender, receiver) = metered_channel::channel_with_total(
+            CORE_THREAD_COMMANDS_CHANNEL_SIZE,
+            &context.metrics.channel_metrics.core_thread,
```

Comment:
I forgot it is still necessary to create separate metrics for metered channels. I can send a PR to use labels instead.